### PR TITLE
Document path conversion on Windows

### PR DESCRIPTION
**Pull request by fredericdalleau**
### Proposed changes

Document how git-bash interacts with the Docker command line on windows.

### CODE REVIEW

**Comment by usha-mandya** [comment]
Made some updates to the format/styling. @fredericdalleau @StefanScherer Could you PTAL and ensure the content still looks good?

**Comment by thaJeztah** [comment]
@fredericdalleau this needs a rebase, because https://github.com/docker/docker.github.io/pull/13353 was merged (I expect `git` will pick up the rename automatically when rebasing)

File: docker-for-windows/troubleshoot.md [hunk 1]
```diff
@@ -157,6 +158,94 @@ For an example of this issue and the resolution, see this issue on GitHub:
...
+In this example, The `$(pwd)` is not converted because of the preceding '/'. However, the second '/work' is transformed by the
+POSIX layer before passing it to Docker Desktop. You can also work around this issue by using an extra `/`.
+
+```console
+$ docker run --rm -ti -v /$(pwd):/work alpine ls //work
+```
+
+To verify whether the errors are generated from your script, or from another source, you can use an environment variable. For example:
+
+```
```
Comment (thaJeztah) [inline docker-for-windows/troubleshoot.md:240]: missed a `console` hint here

**Comment by fredericdalleau** [comment]
I fixed @thaJeztah comment, and spotted a small typo, other than that it looks good!

**Comment by fredericdalleau** [comment]
Oh, and it's rebased

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by usha-mandya** [comment]
Thanks @fredericdalleau LGTM

**Pull request merged**